api_facade.py

```diff
@@ -0,0 +1,290 @@
+            "instructor": "0003",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-256':
+        {
+            "courseName": "Software Process and Project Management",
+            "courseId": "swen-256",
+            "numGrades": 40,
+            "numStudents": 100,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-261':
+        {
+            "courseName": "Introduction to Software Engineering",
+            "courseId": "swen-261",
+            "numGrades": 40,
+            "numStudents": 100,
+            "instructor": "0002",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-262':
+        {
+            "courseName": "Engineering of Software Subsystems",
+            "courseId": "swen-262",
+            "numGrades": 40,
+            "numStudents": 100,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-344':
+        {
+            "courseName": "Web Engineering",
+            "courseId": "swen-344",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-331':
+        {
+            "courseName": "Engineering Secure Software",
+            "courseId": "swen-331",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0002",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-440':
+        {
+            "courseName": "Software System Requirements and Architecture",
+            "courseId": "swen-440",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0002",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-444':
+        {
+            "courseName": "Human Centered Requirements and Design",
+            "courseId": "swen-444",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0001",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-561':
+        {
+            "courseName": "Senior Project I",
+            "courseId": "swen-561",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0003",
+            "students": ["0001", "0002", "0003", "0004"],
+        },
+    'swen-562':
+        {
+            "courseName": "Senior Project II",
+            "courseId": "swen-562",
+            "numGrades": 40,
+            "numStudents": 40,
+            "instructor": "0003",
+            "students": ["9999", "9998", "9997", "9996"],
+        },
+}
+
+
+instructors = {
+    "0001":
+        {
+            "name": "Krutz",
+        },
+    "0002":
+        {
+            "name": "Malachowsky",
+        },
+    "0003":
+        {
+            "name": "Vallino",
+        },
+}
```

Comment:
Similar to what was said above, the api will be expected integers for the id's of the entities.